api/views.py

```diff
@@ -272,15 +273,14 @@ def post(self, request):
         password = request.data.get('password', None)
 
         if username is None or password is None:
-            return bad_request('Body must contain `username` and `password`')
+            return bad_request('Body must contain `email/username` and `password`')
 
         # Get the case-correct username for authenticate()
-        casecorr_uname = User.objects.filter(username__iexact=username).values_list('username', flat=True).first()
-        if casecorr_uname is None:
-            return bad_request('Invalid username or password')  # definitely username issue
+        # casecorr_uname = User.objects.filter(username__iexact=username).values_list('username', flat=True).first()
```

Comment:
can we drop the commented lines of code before merging, @GregoryHorvath ?

Reply:
Done